worldengine/generation.py

```diff
@@ -200,7 +200,7 @@ def generate_world(w, step):
 
     # Prepare sufficient seeds for the different steps of the generation
     rng = numpy.random.RandomState(w.seed)  # create a fresh RNG in case the global RNG is compromised (i.e. has been queried an indefinite amount of times before generate_world() was called)
-    sub_seeds = rng.randint(0, 4294967295, size=100)  # sys.maxsize didn't quite work
+    sub_seeds = rng.randint(0, numpy.iinfo(numpy.int32).max, size=100)  # choose lowest common denominator (32 bit Python seemingly cannot handle a larger value)
```

Comment:
32-bit Python can handle it just fine, it is 32-bit windows Numpy that has issues.
So we go from 4294967295 to 2147483647 which is a real pity. :(